data/jegrami.json

```
@@ -1,6 +1,13 @@
 {
   "name": "Jeremiah Igrami",
-  "bio": "Copyeditor turned technical writer and enthusiastic programmer. I'm passionate about building solutions and telling stories with data",
+  "bio": "**Technical Writer** and **Data Scientist**. I love uncovering insights from huge, messy data.",
```

[Recommendation] I think it’d be great to add some of the statements you had about being a technical writer in your old bio. It’ll highlight both of your interests.
```suggestion
  "bio": "**Technical Writer** and **Data Scientist**. I love uncovering insights from huge, messy data.",
```